infra/modules/worklytics-connectors-msft-365/main.tf

```diff
@@ -68,7 +70,10 @@ resource "local_file" "todo-with-external-todo" {
   for_each = local.todos_to_populate
 
   filename = module.msft_365_grants[each.key].filename
-  content  = format("%s\n## Setup\nThen, please follow next instructions for complete the setup: \n\n%s", module.msft_365_grants[each.key].todo, replace(each.value.external_token_todo, "%%entraid.application_id%%", module.msft_connection[each.key].connector.application_id))
+  content = format("%s\n## Setup\nThen, please follow next instructions for complete the setup: \n\n%s",
+    module.msft_365_grants[each.key].todo,
+    replace(each.value.external_token_todo, "%%entraid.application_ids%%",
```

Comment:
Note that at the moment only Teams has an external to do to follow.

In addition, Teams is a bit particular as it will include the id of the `outlook-cal` app as part of the permissions on TODO. As a result, it will generating something like this on the TODO (just an extract of my local):

>  4. Follow steps on [Configure application access to online meetings or virtual events]> (https://learn.microsoft.com/en-us/graph/cloud-communication-online-meeting-application-access-policy):
 > - Add a policy for the application created for the connector, providing its `application id` (NOTE: if `outlook-cal` is already present, it will be included as well it contains the OnlineMeetings role)
>```shell
>New-CsApplicationAccessPolicy -Identity Teams-Policy-For-Worklytics -AppIds "teams app id >redacted","outlook-cal app id redacted" -Description "Policy for MSFT Teams used for Worklytics Psoxy >connector"
>```
>  - Grant the policy to the whole tenant (NOT to any specific application or user)
>```shell
>Grant-CsApplicationAccessPolicy -PolicyName Teams-Policy-For-Worklytics -Global
>```

As `New-CsApplicationAccessPolicy` supports multiple app ids on the parameter and the scope of this is *teams*, I think better to have it here instead of providing to outlook-cal its own TODO for doing exactly the same but with its id. With this, in a single set of instructions and commands both can be enabled.